big-int/src/bigint.hpp

```diff
@@ -50,6 +50,76 @@ class largeInt
     explicit largeInt(long long int);
     explicit largeInt(std::string);
 
+    std::string getnumber(){
+        return number;
+    }
+    
+    int multiply(int x, int res[], int res_size) {
+        int carry = 0;
+    
+        for (int i = 0; i < res_size; i++) {
+            int prod = res[i] * x + carry;    
+            res[i] = prod % 10;
+            carry = prod / 10;
+        }
+    
+        while (carry) {
+            res[res_size] = carry % 10;
+            carry = carry / 10;
+            res_size++;
+        }
+        return res_size;
+    }
+    
+    largeInt* pow(int a, int b) 
```

Comment:
here we are not dealing with `int` data type, you are supposed to write a pow() fuction for large integers. and also the way you are working is not correct, headerfiles are supposed to contain only the function declarations not the definitions.